app/views/clearance/sessions/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_for :session, url: session_path do |form| %>
+  <div class="input-group">
+  <span class="input-group-addon" id="basic-addon1">E-mail</span>
+    <%= form.text_field :email %>
+  </div>
+
+  <div class="input-group">
+  <span class="input-group-addon" id="basic-addon1">Password</span>
+    <%= form.text_field :password %>
+  </div>
+
+  <div class="submit-field">
+    <%= form.submit %>
+  </div>
+
+<% end %>
```

Comment:
Please configure your editor to insert a newline at the end of the file automatically. If you're using Sublime, the configuration option is named: `ensure_newline_at_eof_on_save`.